content-warehouse/src/main/java/contentwarehouse/v1/QuickStart.java

```diff
@@ -0,0 +1,142 @@
+import com.google.cloud.contentwarehouse.v1.DocumentSchemaServiceClient;
+import com.google.cloud.contentwarehouse.v1.DocumentSchemaServiceSettings;
+import com.google.cloud.contentwarehouse.v1.DocumentServiceClient;
+import com.google.cloud.contentwarehouse.v1.DocumentServiceSettings;
+import com.google.cloud.contentwarehouse.v1.LocationName;
+import com.google.cloud.contentwarehouse.v1.Property;
+import com.google.cloud.contentwarehouse.v1.PropertyDefinition;
+import com.google.cloud.contentwarehouse.v1.RequestMetadata;
+import com.google.cloud.contentwarehouse.v1.TextArray;
+import com.google.cloud.contentwarehouse.v1.TextTypeOptions;
+import com.google.cloud.contentwarehouse.v1.UserInfo;
+import com.google.cloud.resourcemanager.ProjectInfo;
+import com.google.cloud.resourcemanager.ResourceManager;
+import com.google.cloud.resourcemanager.ResourceManagerOptions;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+public class QuickStart {
+  
+  public static void main(String[] args)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String location = "your-region"; // Format is "us" or "eu".
+    String userId = "your-user-id"; // Format is user:<user-id>
+    quickStart(projectId, location, userId);
+  }
+
+  public static void quickStart(String projectId, String location, String userId)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    String projectNumber = getProjectNumber(projectId);
+
+    String endpoint = String.format("%s-contentwarehouse.googleapis.com:443", location);
+    DocumentSchemaServiceSettings documentSchemaServiceSettings = 
+         DocumentSchemaServiceSettings.newBuilder().setEndpoint(endpoint).build(); 
+  
+    // Create a Schema Service client
+    try (DocumentSchemaServiceClient documentSchemaServiceClient =
+        DocumentSchemaServiceClient.create(documentSchemaServiceSettings)) {
+      /*  The full resource name of the location, e.g.:
+      projects/{project_number}/locations/{location} */
+      String parent = LocationName.format(projectNumber, location);
+
+      /* Create Document Schema with Text Type Property Definition
+       * More detail on managing Document Schemas: 
+       * https://cloud.google.com/document-warehouse/docs/manage-document-schemas */
+      DocumentSchema documentSchema = DocumentSchema.newBuilder()
+          .setDisplayName("My Test Schema")
+          .setDescription("My Test Schema's Description")
+          .addPropertyDefinitions(
+            PropertyDefinition.newBuilder()
+            .setName("stock_symbol")
+            .setDisplayName("Searchable text")
+            .setIsSearchable(true)
+            .setTextTypeOptions(TextTypeOptions.newBuilder().build())
```

Comment:
Not sure why the lint didn't catch this, but it would be nice for this to be indented.

Reply:
That is weird. I made this change, reran mvn checkstyle and got no violations. Will update this